src/includes/components/sidebar.php

```diff
@@ -8,77 +8,113 @@
+				<div class="submenu-item <?php echo basename($_SERVER['PHP_SELF']) == 'manage-candidate.php' ? 'active' : ''; ?>"
+					data-bs-toggle="collapse" href="#manageCandidates" id="submenuToggle" data-bs-parent="false">
+					<div class="submenu-content">
+						<i data-feather="user-plus" class="white mb-xl-1"></i>
+						<span style="padding-left: 0.7rem;">Candidates</span>
+					</div>
+					<i class="fas fa-chevron-right" id="submenuIcon"></i>
+				</div>
+				<ul class="menu-items submenu collapse" id="manageCandidates">
+					<li class="item">
+						<a href="manage-acc.php"
+							class="<?php echo basename($_SERVER['PHP_SELF']) == 'add-candidate.php' ? 'active-sub fw-bold' : ''; ?>">Add Candidate</a>
+					</li>
+					<li class="item">
+						<a href="admin-creation.php"
+							class="<?php echo basename($_SERVER['PHP_SELF']) == 'manage-candidate.php' ? 'active-sub fw-bold' : ''; ?>">Manage Candidates</a>
+					</li>
+				</ul>
 			</li>
 
 			<li class="item">
-				<a href="archive.php"><i data-feather="archive" class="white me-xl-3 mb-xl-1"></i>Archive</a>
+				<a href="archive.php"
+					class="<?php echo basename($_SERVER['PHP_SELF']) == 'archive.php' ? 'active' : ''; ?>"><i
+						data-feather="archive" class="white mb-xl-1"></i><span
+						style="padding-left: 1rem">Archive</span></a>
 			</li>
 
 			<li class="item">
-				<a href="configuration.php"><i data-feather="settings" class="white me-xl-3 mb-xl-1"></i>Configuration</a>
+				<a href="configuration.php"
+					class="<?php echo basename($_SERVER['PHP_SELF']) == 'configuration.php' ? 'active' : ''; ?>"><i
+						data-feather="settings" class="white mb-xl-1"></i><span
+						style="padding-left: 1rem">Configuration</span></a>
 			</li>
 		</ul>
 	</div>
 </nav>
 
+
 <nav class="navbar">
 	<div class="container-fluid">
-		<div class="header-left">
-			<i class="fas fa-bars" id="sidebar-close"></i>
+		<div class="header-left main-color">
+			<i class="fas fa-bars " id="sidebar-close"></i>
 		</div>
 
 		<div>
 			<img src="images/resc/ivote-logo.png" class="me-xl-3" style="height:35px">
 		</div>
 
 		<div class="header-right">
-			<div class="dropdown">
+			<div class="dropdown user-profile">
+
 				<button class="btn" id="dropdownMenuButton" data-bs-toggle="dropdown" aria-expanded="false">
-					<img src="images/logos/jpia.png" class="profile-icon me-xl-3"><i
+					<img src="images/logos/<?php echo $org_name ?>.png" class="profile-icon me-xl-3"><i
 						class="fas fa-chevron-down main-color fs-6"></i>
 				</button>
-				<ul class="dropdown-menu dropdown-menu-end" aria-labelledby="dropdownMenuButton">
+				<ul class="dropdown-menu dropdown-menu-end main-color" aria-labelledby="dropdownMenuButton">
 					<li><a class="dropdown-item" href="#">Profile</a></li>
-					<li><a class="dropdown-item" href="#">Log Out</a></li>
+					<li><a class="dropdown-item" href="voter-logout.php"">Log Out</a></li>
```

Comment:
The excess double quotation marks might cause problems in the long run. Kindly fix this.